components/more_direct_channels/index.js

```diff
@@ -66,7 +66,7 @@ function mapStateToProps(state, ownProps) {
         currentTeamId: team.id,
         currentTeamName: team.name,
         searchTerm,
-        users,
+        users: users.sort(sortByUsername),
```

Comment:
I believe this will create a new array every time and will cause unnecessary re-renders every time anything in the store changes, you may want to create a selector for this.

Reply:
👍

Reply:
actually nvm

> The sorted array. Note that the array is sorted in place, and no copy is made.

![image](https://user-images.githubusercontent.com/6757047/68771018-33a81000-05f5-11ea-80e6-936b2abf3c6b.png)

Reply:
:)